### PR TITLE
feat: add "received at" timestamp to invites

### PR DESCRIPTION
**Pull request by EvanHahn**
This will make it possible for the frontend to sort incoming invites, and was requested by Erik.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -90,7 +98,7 @@
     "eslint": "^8.57.0",
     "filter-obj": "^6.0.0",
     "husky": "^8.0.0",
-    "iterpal": "^0.3.0",
+    "iterpal": "^0.4.0",
```
Comment (EvanHahn) [inline package.json:101]: Necessary to pull in `compact` for tests.

File: package.json [hunk 2]
```diff
@@ -49,7 +49,15 @@
       "ecmaVersion": 13,
       "sourceType": "module"
     },
-    "rules": {}
+    "rules": {
+      "no-unused-vars": [
+        "error",
+        {
+          "varsIgnorePattern": "^_",
+          "argsIgnorePattern": "^_"
```
Comment (EvanHahn) [inline package.json:57]: I use `varsIgnorePattern` in this PR, but not `argsIgnorePattern`. I added it for consistency.

File: tests/invite-api.js [hunk 3]
```diff
@@ -1047,3 +1099,73 @@ function setup() {
     encryptionKeys,
   }
 }
+
+/**
+ * Assert that invites are alike, allowing for some wiggle room around
```
Comment (gmaclennan) [inline tests/invite-api.js:1104]: can you add a note about what wiggle room means here, to help a future maintainer who might find this failing.

**Review by gmaclennan — APPROVED** [review]
lots of changes to the tests for a small change in the code! Looks ok though. Not sure about the eslint rule - I prefer to ignore on a case-by-case, but ok for this repo since you'll be taking the lead.

File: tests/invite-api.js (hunk 3 above)
Comment (EvanHahn) [reply tests/invite-api.js]: Done in 8229dacf9a3ac9db495180ecea002ce83fb0e67a.

**Pull request merged**